src/main/java/com/znsio/e2e/tools/Driver.java

```diff
@@ -221,6 +222,14 @@ public void bringAppInForeground () {
         ((StartsActivity) driver).currentActivity();
     }
 
+    public void deepLinkingAUrl(String url, String packageName) {
```

Comment:
The name of the method does not indicate what is really being done. Is it setting up a deep linking URL?

Reply:
updated